File: CrimsonClubs/CrimsonClubs.Models/Dtos/JoinRequestDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using CrimsonClubs.Models.Entities;

namespace CrimsonClubs.Models.Dtos
{
    public class JoinRequestDto
    {
        public int UserId { get; set; }
        public string First { get; set; }
        public string Last { get; set; }
        public JoinRequestDto(MM_User_Club relation)
        {
            var user = relation.User;

            UserId = user.Id;
            First = user.First;
            Last = user.Last;
        }
    }
}

File: Android/CrimsonClubs/app/src/main/java/crimsonclubs/uacs/android/crimsonclubs/JoinRequestDto.java
package crimsonclubs.uacs.android.crimsonclubs;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class JoinRequestDto {

    @Expose
    @SerializedName("userId")
    public int userId;
    @Expose
    @SerializedName("first")
    public String first;
    @Expose
    @SerializedName("last")
    public String last;


}
